content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -353,6 +352,22 @@ The `PodTopologySpread` plugin does not score the nodes that don't have
 the topology keys specified in the spreading constraints.
 {{< /note >}}
 
+If you don't want to use the default Pod spreading constraints for your cluster,
+you can disable those defaults by setting `defaultingType` to `List` and leaving
+empty `defaultConstraints` in the `PodTopologySpread` plugin configuration:
+
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1beta1
+kind: KubeSchedulerConfiguration
+
+profiles:
+  - pluginConfig:
+      - name: PodTopologySpread
+        args:
+          defaultConstraints: []
```

Comment:
@alculquicondor What if user don't specify `args.defaultConstraints` while specifying "defaultingType: List"? I think it also works, right?

Reply:
It does work. I just went with the configuration that it's more explicit and easier to explain.